napalm_logs/base.py

```diff
@@ -40,14 +40,6 @@
 
 class NapalmLogs:
     def __init__(self,
-                 address='0.0.0.0',
```

Comment:
The reason that I removed these is because for the plugable listener and publisher modules you do not necessarily require these. For instance for the kafka modules you should pass a list of hosts, or (yet to add but preferred) use a SRV record to discover the bootstrap server(s).